Gems/Prefab/PrefabBuilder/PrefabGroup/tests/PrefabBehaviorTests.cpp

```diff
@@ -410,4 +428,79 @@ namespace UnitTest
         EXPECT_EQ(result, ProcessingResult::Ignored);
         EXPECT_EQ(scene->GetManifest().GetEntryCount(), 0);
     }
+
+    TEST_F(PrefabBehaviorTests, PrefabBehavior_IgnoreActors_ToggleTrueReturnsIgnored)
```

Comment:
Awesome work adding the tests! Super helpful to detect the logic is working as expected